http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -327,5 +354,9 @@ final case class AKSInterpreterConfig(terraAppSetupChartConfig: TerraAppSetupCha
                                       coaAppConfig: CoaAppConfig,
                                       aadPodIdentityConfig: AadPodIdentityConfig,
                                       appRegistrationConfig: AzureAppRegistrationConfig,
-                                      samConfig: SamConfig
+                                      samConfig: SamConfig,
+                                      pollingConfig: PollingConfig =
+                                        PollingConfig(maxAttempts = 120, delay = 10 seconds, interruptAfter = 20 minutes)
```

Comment:
Does this seem alright or should I expose these as environment variables/.conf values?